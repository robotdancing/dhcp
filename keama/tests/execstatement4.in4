# DHCPv4 executable statement config

# authoritative is mandatory
authoritative;

# empty configs are not accepted by Kea
default-lease-time 1800;

# impossible to convert statement statement
break;
