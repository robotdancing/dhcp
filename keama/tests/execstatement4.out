{
  "Dhcp4": {
    "valid-lifetime": 1800
//  # impossible to convert statement statement
//  "statement": {
//    "break": null
//  }
  }
}
